.travis/custom_release.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+NODE_ENV=production npm run build
+
+if [ "${TRAVIS_BRANCH}" = "master" ]
+then
+    for env in ci qa
+    do
+        echo "PUSHING ${env}-beta"
+        rm -rf ./dist/.git
```

Comment:
Where does this come from? I've never seen a git directory in a build directory.

Reply:
The platform team pointed us to this example which this PR is largely a copy of: https://github.com/RedHatInsights/landing-page-frontend/blob/master/.travis/custom_release.sh.

For reference, the `release.sh` script that is called lives here: https://github.com/RedHatInsights/insights-frontend-builder-common/tree/master/src

Reply:
Ok. It just surprised me a bit. It is probably a failsafe or something.

Reply:
Without this line the changes only get deployed to the first static repo encountered in this loop.  At least that is what it looked like when we excluded this line from the sources-ui release script.  I haven't had time to dig into it more.